src/components/storeWrapper.js

```diff
@@ -0,0 +1,16 @@
+export default function wrapStore(providedStore, ngReduxStore) {
+  const unsubscribe = providedStore
+    .subscribe(() => {
+      let newState = providedStore.getState();
+
+      ngReduxStore.dispatch(newState);
```

Comment:
here's where the magic happens. The only thing I'm unsure about is cleanup. But as I understand it, Angular 1 never really "unloads" during runtime so this shouldn't be an issue.